packages/playground/blueprints/src/lib/steps/enable-multisite.spec.ts

```diff
@@ -36,13 +36,21 @@ describe('Blueprint step enableMultisite', () => {
 		});
 		await enableMultisite(php, {});
 
+		/**
+		 * Relying on HTTP requests for unit tests won't work because of the wp-login redirects.
```

Comment:
can this comment clarify how the redirects prevent the unit test from working? So that someone with no knowledge about this discussion would immediately understand